File: LED.c
#include <stdio.h>
#include <string.h>

int main() {
    int ledsPorNumero[10] = {6, 2, 5, 5, 4, 5, 6, 3, 7, 6};

    int n; 
    scanf("%d", &n); 

    while (n--) {
        char valor[10100]; 
        scanf("%s", valor); 

        int totalLeds = 0;

       
        for (int i = 0; i < strlen(valor); i++) {
            totalLeds += ledsPorNumero[valor[i] - '0'];
        }

        printf("%d leds\n", totalLeds);
    }

    return 0;
}

File: LED.py
leds_por_numero = {
    '0': 6,
    '1': 2,
    '2': 5,
    '3': 5,
    '4': 4,
    '5': 5,
    '6': 6,
    '7': 3,
    '8': 7,
    '9': 6
}

n = int(input())

for _ in range(n):
    valor = input()  
    total_leds = sum(leds_por_numero[d] for d in valor)  
    print(f"{total_leds} leds")